src/aws/config.rs

```diff
@@ -90,6 +90,10 @@ pub struct Config {
     /// - this field if it's `is_some`.
     /// - profile config: `external_id`
     pub external_id: Option<String>,
+    /// `tags` value will be loaded from:
+    ///
+    /// - this field if it's `is_some`
+    pub tags: Option<Vec<String>>,
```

Comment:
If tags must be a key-value, how about store them as `Vec<(String, String)>` instead? We can avoid to split them again.

Reply:
Yeah, that should make it easier. Made the suggested changes